client/src/modules/stock/settings/stock-settings.js

```diff
@@ -43,6 +43,14 @@ function StockSettingsController(
           .catch(Notify.handleError);
       })
       .catch(Notify.handleError);
+
+    // load algorithmes for Average Consumption
+    StockSettings.getAverageConsuptionAlgo()
```

Comment:
I find it hard to keep track of which algorithm is 1, 2, etc.   I suggest appending the algorithm number on these functions:    getAvertageConsomptionAlgo1()     (add the correct number!)
That way is is easier to track what is happening.

Reply:
This function just allows you to obtain the different existing algorithms...
````
// GET '/stock/setting/average_consuption'
// Get the different algorithms for calculating the average monthly consumption
exports.averageConsuptionAlgo = function averageConsuptionAlgo(req, res, next) {
  const sql = `SELECT id, name, label, comment FROM average_consumption_algo;`;

  db.exec(sql)
    .then((rows) => {
      res.status(200).json(rows);
    })
    .catch(next)
    .done();
};

````

Reply:
Ok.  Thx.